saul_coap.c

```diff
@@ -13,7 +13,8 @@
  * @file
  * @brief       CoAP endpoint for the SAUL registry
  *
- * @author      Micha Rosenbaum <micha.rosenbaum@haw-hamburg.de>
+ * @author      Seojeong Moon <seojeong.moon@haw-hamburg.de>
+   @author Micha Rosenbaum <micha.rosenbaum@haw-hamburg.de>
```

Comment:
Thank you, that you included yourself. I should have done this earlier.

Could need some code style improvements, like the `*` at the beginning and the spaces between the `@author` tag and the name.